R/bart.R

```diff
@@ -1,47 +1,84 @@
 
 #' fitBARTSurvival
 #'
-#' @param trainXs A matrix/data.frame of X variables to be used in the model fitting. Columns should correspond to variables.
-#' @param trainTarget A matrix/data.frame including columns named 'Event' and 'time_to_event' corresponding to a binary indicator of event/censoring and the time to event/censoring respectively.
-#' @param testXs A matrix/data.frame of out-of-sample data to be used in model evaluation. The set of variables should be the same as those in trainXs.
-#' @param pipelineRunInformation The object originally created by begin.pipeline.run.
-#' @param sparse Set to TRUE if the sparsity inducing Dirichlet prior should be used for splitting rule variable selection.
+#' @param trainXs A matrix/data.frame of X variables to be used in the model
+#'   fitting. Columns should correspond to variables.
+#' @param trainTarget A matrix/data.frame including columns named 'Event' and
+#'   'time_to_event' corresponding to a binary indicator of event/censoring and
+#'   the time to event/censoring respectively.
+#' @param testXs A matrix/data.frame of out-of-sample data to be used in model
+#'   evaluation. The set of variables should be the same as those in
+#'   \code{trainXs}.
+#' @param pipelineRunInformation The object originally created by
+#'   \code{\link{beginPipelineRun}}.
```

Comment:
\link{} allows hyperlinks in your documentation to the documentation of other functions. You can also use \link[<PKG>]{<function>} to link to the documentation in another package